components/teach-like-mozilla-page.jsx

```diff
@@ -1,18 +1,97 @@
 var React = require('react');
 
 var HeroUnit = require('./hero-unit.jsx');
+var IconLink = require('./icon-link.jsx');
+
+var IconLinks = React.createClass({
+  ICON_LINKS: [
+    {
+      linkTo: "fixme",
```

Comment:
Before merging this, can you make sure to create an issue to update these links?